packages/cli/src/commands/labs/index.ts

```diff
@@ -5,18 +5,23 @@ import {ux} from '@oclif/core'
 import {sortBy} from 'lodash'
 
 interface Features {
-  currentUser: any,
-  user: any,
-  app?: any,
+  currentUser: Heroku.Account,
+  user: Heroku.AccountFeature,
+  app?: Heroku.AppFeature | null,
 }
 
 function printJSON(features: Heroku.Account | Heroku.AccountFeature | Heroku.AppFeature) {
   ux.log(JSON.stringify(features, null, 2))
 }
 
-function printFeatures(features: any) {
+function printFeatures(features: Heroku.AppFeature[] | Heroku.AccountFeature | null) {
   features = sortBy(features, 'name')
   const longest = Math.max(...features.map((f: Record<string, string>) => f.name.length))
```

Comment:
When `Heroku.AccountFeature` or `null` is passed in, a call to `map` will fail unless `Heroku.AccountFeature` specifies a `CallableFunction` type on the `map` property.